### ISSUE TITLE
 How to load the trained model as a starting point for the new training with other classes and prompts?

### ISSUE THREAD

**Issue by dmityul** [TheLastBen/fast-stable-diffusion#90]
I did this with another colab (https://colab.research.google.com/github/ShivamShrirao/diffusers/blob/main/examples/dreambooth/DreamBooth_Stable_Diffusion.ipynb), just changed initial model path, but I don't understand how to point this fast-stable-diffusion colab to the file I made with it. (I'm not a programmer)

**Comment by TheLastBen** [comment]
You can not use a CKPT for the training, the colab you pointed let's you choose a path to a diffusers model, not a ckpt, you need to convert your trained model into diffusers, then retrain it, but I doubt it will keep the quality

**Comment by dmityul** [comment]
Fast-stable-diffusion diffusion saves the trained model only as a ckpt?

**Comment by TheLastBen** [comment]
you can recover the trained diffusers model in the file explorer, but if there many requests for it, i'll add an option for it

**Comment by C-D-Harris** [comment]
It breaks at the end of training when using the trained diffusers as the starting model because it's looking for the feature_extractor and safety_checker but they were discarded in the previous run.

**Comment by TheLastBen** [comment]
@C-D-Harris You can still add those from the default model

**Comment by jujpenabe** [comment]
I have converted a ckpt to a diffuser model (.bin), to retrain DB with this and not the default sd V1.4, do I just need to point to the path of the converted file?

It would be useful to have an option to get the diffuser model after training.

**Comment by TheLastBen** [comment]
Yes you just need to change the path in the cell setting up MODEL_NAME.
You can find the diffusers model in the folder models using the colab file explorer

**Comment by jujpenabe** [comment]
I replaced MODEL_NAME="/content/stable-diffusion" with the path of my .bin model, something like "/content/gdrive/MyDrive/IA/Models/diffusion_pytorch_model.bin" but it failed on Start Dreambooth Cell. This variable is only used in `--pretrained_model_name_or_path` on line 45, right?, so maybe that's the problem.

On the other hand, just below the `train_dreambooth.py` command in line 65 the program exits `if os.path.exists('/content/models/'+INSTANCE_NAME+'/unet/diffusion_pytorch_model.bin'):`
 I checked and is true that /content/models doesnt exists. I dont know when is that dir created ( maybe when `train_dreambooth.py` is running?)

**Comment by TheLastBen** [comment]
the path should be to the folder containing the model folders (vae, unet ...etc)
